### PR TITLE
Latest version of ui.R and Server.R

### PR DESCRIPTION
**Pull request by chaninn**

### CODE REVIEW

File: App-1/ui.R [hunk 1]
```diff
@@ -1,35 +1,90 @@
 library(shiny)
-library(shinythemes)
+library(seqinr)
 library(protr)
-library(markdown)
+library(caret)
+library(randomForest)
 
+Train_DPC_PCP <- read.csv("Train_DPC_PCP.csv", header=TRUE)
```
Comment (sawsimeon) [inline App-1/ui.R]: Boss, i am just wondering is the code for shiny ui or server boss.

Comment (sawsimeon) [inline App-1/ui.R]:
```
shinyServer(function(input, output, session) {
```

Comment (chaninn) [inline App-1/ui.R]: My bad, its the contents for Server indeed, llet me re-do the commits
and pull requests.

On Tuesday, May 12, 2015, Saw Simeon notifications@github.com wrote:

> In App-1/ui.R
> https://github.com/Rnewbie/OliFP/pull/7#discussion_r30101576:
> 
> > +Train_DPC_PCP <- read.csv("Train_DPC_PCP.csv", header=TRUE)
> 
> shinyServer(function(input, output, session) {
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/Rnewbie/OliFP/pull/7/files#r30101576.

## 

Associate Professor Chanin Nantasenamat, Ph.D.
Head, Center of Data Mining and Biomedical Informatics
Faculty of Medical Technology, Mahidol University
999 Phutthamonthon 4 Road, Salaya, Nakhon Pathom 73170
Tel. 02 441 4371 ext 2715
Fax 02 441 4380

**Comment by sawsimeon** [comment]
Boss, I don't see the error message boss.

**Pull request merged**